appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -1253,7 +1255,9 @@ private YaVersion() {
 
   // For CLOUDDB_COMPONENT_VERSION 1:
   // - CloudDB component introduced
-  public static final int CLOUDDB_COMPONENT_VERSION = 1;
+  // - Added the UpdateDone event and ProvideUpdateDone property
```

Comment:
To be consistent with the other components, include `// For CLOUDDB_COMPONENT_VERSION 2:`

Reply:
Oops. Fixed this in the latest commit